packages/sdk-react/src/components/auth/TurnkeyThemeProvider.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { createContext, useContext, useEffect } from "react";
```

Comment:
We should add `"use client"` here so devs don't need to wrap it or use `"use client"` in the root layout